osu.Game.Tests/NonVisual/Skinning/LegacySkinTextureFallbackTest.cs

```diff
@@ -127,8 +127,29 @@ public void TestReturnNullOnFallbackFailure()
             Assert.IsNull(texture);
         }
 
+        [Test]
```

Comment:
Can you also add a test testing the inverse behaviour?

Reply:
This is supposed to be handled by most test cases above, but I'll add one for the sake of being explicit.